### PR TITLE
Content: Latest Dashboard 2.0 updates

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

New article to talk about the latest and greatest features that have been added to Dashboard in `v.1.8.0`

Still need a [relevant tile](https://github.com/FlowFuse/website/issues/1974)to support the article

### CODE REVIEW

**Comment by joepavitt** [comment]
@Hasmin-AC added for SEO review per guidance in Handbook

File: src/blog/2024/04/dashboard-1-8-0.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
+---
+title: "Dashboard 2.0: Milestones, PWA and New Components"
```
Comment (Hasmin-AC) [inline src/blog/2024/04/dashboard-1-8-0.md:2]: Is PWA a well enough known term? Should we expand on it in the title?

**Comment by Hasmin-AC** [comment]
An idea for the title - Progressive Web App  support and more in latest Dashboard 2.0 release

File: src/blog/2024/04/dashboard-1-8-0.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    - Vuetify
+---
+
+With a new release of Node-RED Dashboard 2.0 we have plenty of new fixes and improvements being added to the project. In this post, we'll deep dive into community contributions, PWA support, new Vuetify components, and the rest of the great work published in this latest release.
+
+<!--more-->
+
+## Community Contributions
+
+I firstly wanted to take this opportunity to point out a big milestone that I'm very proud to see in Node-RED Dashboard 2.0.
```
Comment (ZJvandeWeg) [inline src/blog/2024/04/dashboard-1-8-0.md:26]:
```suggestion
We firstly wanted to take this opportunity to point out a big milestone that we're very proud to see in Node-RED Dashboard 2.0.
```

**Comment by joepavitt** [comment]
> An idea for the title - Progressive Web App  support and more in latest Dashboard 2.0 release 

The new Vuetify components piece will draw more clicks imo, but PWA is a very close second.

File: src/blog/2024/04/dashboard-1-8-0.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+So thank you very much [@BartButenaers](https://github.com/bartbutenaers), [@Ek1nox](https://github.com/Ek1nox), [@fullmetal-fred](https://github.com/fullmetal-fred) and [@cgjgh](https://github.com/cgjgh) for for great efforts and initiative in improving Node-RED Dashboard 2.0.
+
+For anyone else that's interested in contributing to the project, please do reach out, and we'll be happy to help you get started. We also have a [Contributing Guide](https://dashboard.flowfuse.com/contributing/) if you want to dive straight in.
+
+## Progressive Web App (PWA) Support
+
+The biggest community contribution we saw in this release was the addition of Progressive Web App (PWA) support. This feature was added by [@cgjgh](https://github.com/cgjgh), and allows you to install your Node-RED Dashboard 2.0 applications directly onto your platform, including Windows, iOS and Android.
+
+This work will give your Dashboard's a much more native/natural feel when running on your own machines, and mean you no longer need to go via your browser to access your applications.
```
Comment (ZJvandeWeg) [inline src/blog/2024/04/dashboard-1-8-0.md:38]: Link to docs on how to achieve this and use it?

File: src/blog/2024/04/dashboard-1-8-0.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+## New Vuetify (Preview) Components Available
+
+Vuetify is the component library on which most of our Dashboard 2.0 components are built. Our core widgets implement the more fundamental UI elements, but that doesn't stop you from building out fully customized interfaces yourself using our [ui-template node](https://dashboard.flowfuse.com/nodes/widgets/ui-template.html) with the vast collection of Vuetify components.
+
+Within the `ui-template` node, we natively support any of the [core Vuetify components](https://vuetifyjs.com/en/components/all/#containment), but Vuetify itself is always evolving and often they release components into their [Vuetify Labs](https://vuetifyjs.com/en/labs/introduction/#what-is-labs).
```
Comment (ZJvandeWeg) [inline src/blog/2024/04/dashboard-1-8-0.md:44]: I feel this is a long winded way of saying we've updated Vuetify. Not sure why Vuetify Labs are explained -- it's very much a tech detail that's more documentation than a release post?

**Comment by joepavitt** [comment]
> I think it's well known, might not be well enough though. I'd rephrase the title and filename (for SEO) to:

Not sure about the `v1.8` piece - it's already confusing enough with 2.0 being referenced, and I've not explicitly called out version numbers in the titles before

File: src/blog/2024/04/dashboard-1-8-0.md (hunk 1 above)
Comment (joepavitt) [reply src/blog/2024/04/dashboard-1-8-0.md:2]: Not sure about the v1.8 piece - it's already confusing enough with 2.0 being referenced, and I've not explicitly called out version numbers in the titles before

File: src/blog/2024/04/dashboard-1-8-0.md (hunk 3 above)
Comment (joepavitt) [reply src/blog/2024/04/dashboard-1-8-0.md:38]: We actually don't have any currently, I'll see what I can put together.

File: src/blog/2024/04/dashboard-1-8-0.md (hunk 4 above)
Comment (joepavitt) [reply src/blog/2024/04/dashboard-1-8-0.md:44]: wanted to call it out as the Vuetify Labs components aren't stable - APIs and properties can change, etc, so wanted to make sure it was clear to Dashboard users which ones are "labs" and which ones are "core"

File: src/blog/2024/04/dashboard-1-8-0.md (hunk 1 above)
Comment (joepavitt) [reply src/blog/2024/04/dashboard-1-8-0.md:2]: "New Components" makes the title more clickbait-y and more likely to have users click to find out what the components are, if we put too much into the title, no reason to engage

File: src/blog/2024/04/dashboard-1-8-0.md (hunk 4 above)
Comment (ZJvandeWeg) [reply src/blog/2024/04/dashboard-1-8-0.md:44]: Fair enough

File: src/blog/2024/04/dashboard-1-8-0.md (hunk 1 above)
Comment (ZJvandeWeg) [reply src/blog/2024/04/dashboard-1-8-0.md:2]: Can you move the file to a path that matches the title at least?

**Review by ZJvandeWeg — APPROVED** [review]
Pending the tile, I approve pre-emptively though.
